lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -73,19 +84,51 @@ def before_scenario_starts(scenario)
       end
 
       def before_execution_starts(execution)
-        @spinner.update(execution_info(execution, 'running'))
-        @spinner.activate
+        puts(execution_info(execution, ''))
+      end
+
+      def print(string)
+        new_line_if_needed
+        @stdout.print(string)
+        @stdout.flush
+        record_last_line(string)
       end
 
-      def on_execution_update(execution, update)
```

Comment:
@iNecas I think I missed this in review.
But [`Execution#update`](https://github.com/iNecas/foreman_maintain/blob/master/lib/foreman_maintain/runner/execution.rb#L55) is still calling `on_execution_update`. Origin seems to be `say` method. It looks like if you are **not** opting for the spinner and calling say then it raises an exception
```undefined method `on_execution_update' for #<ForemanMaintain::Reporter::CLIReporter:0x00000001eb5900>```